pkg/webui/account/views/forgot-password/index.js

```diff
@@ -58,81 +55,66 @@ const initialValues = { user_id: '' }
+    async (values, { resetForm, setSubmitting }) => {
+      try {
+        setError(undefined)
+        await tts.Users.createTemporaryPassword(values.user_id)
+        sendToLogin()
+      } catch (error) {
+        setSubmitting(false)
+        setError(error)
+      }
+    },
+    [sendToLogin],
+  )
 
-  @bind
-  async handleSubmit(values, { setSubmitting }) {
-    try {
-      await api.users.resetPassword(values.user_id)
-      this.setState({
-        error: '',
-        info: m.passwordRequested,
-        requested: true,
-      })
-    } catch (error) {
-      this.setState({
-        error,
-        info: '',
-      })
-    } finally {
-      setSubmitting(false)
-    }
-  }
-
-  render() {
-    const { error, info, requested } = this.state
-    const { handleCancel } = this.props
-    const cancelButtonText = requested ? m.goToLogin : sharedMessages.cancel
-
-    return (
-      <div className={style.form}>
-        <IntlHelmet title={m.forgotPassword} />
-        <h1 className={style.title}>
-          {siteName}
-          <br />
-          <Message component="strong" content={m.requestTempPassword} />
-        </h1>
-        <hr className={style.hRule} />
-        <Message content={m.resetPasswordDescription} component="p" className={style.description} />
-        <Form
-          onSubmit={this.handleSubmit}
-          initialValues={initialValues}
-          error={error}
-          info={info}
-          validationSchema={validationSchema}
-          horizontal={false}
-        >
-          <Form.Field
-            title={sharedMessages.userId}
-            name="user_id"
-            component={Input}
-            autoFocus
-            required
-          />
-          <Form.Submit
-            component={SubmitButton}
-            message={m.send}
-            className={style.submitButton}
-            alwaysEnabled
-          />
-          <Button naked secondary message={cancelButtonText} onClick={handleCancel} />
-        </Form>
-      </div>
-    )
-  }
+  return (
+    <div className={style.form}>
+      <IntlHelmet title={m.forgotPassword} />
+      <h1 className={style.title}>
+        {siteName}
+        <br />
+        <Message component="strong" content={m.resetPassword} />
+      </h1>
+      <hr className={style.hRule} />
+      <Message content={m.resetPasswordDescription} component="p" className={style.description} />
+      <Form
+        onSubmit={handleSubmit}
+        initialValues={initialValues}
+        error={error}
+        validationSchema={validationSchema}
+      >
+        <Form.Field
+          title={sharedMessages.userId}
+          name="user_id"
+          component={Input}
+          autoFocus
+          required
+        />
+        <Form.Submit
+          component={SubmitButton}
+          message={m.send}
+          className={style.submitButton}
+          alwaysEnabled
+        />
+        <Button.Link naked secondary message={sharedMessages.cancel} to="/login" />
```

Comment:
1. Navigate to `/console`
2. Land on the login page and go to the forgot password view
3. Submit or just navigate back
4. Login
5. Land on the account app overview instead of the Console

Not sure if we should preserve the initial users location, but is seems weird that we do that for registration and not for the forgot password view.